stable/dex/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: dex
-version: 0.8.0
+version: 0.8.1
```

@michaelajr lets bump to 1.0.0 since this chart is in the stable folder
```suggestion
version: 1.0.0
```